common/web/types/src/ldml-keyboard/ldml-keyboard-xml.ts

```diff
@@ -24,11 +24,11 @@ export interface LKKeyboard {
   names?: LKNames;
   settings?: LKSettings;
   keys?: LKKeys;
+  displays?: LKDisplays;
   layers?: LKLayerMaps[];
   vkeys?: LKVkeyMaps;
   transforms?: LKTransforms[];
   reorders?: LKReorders;
-  backspaces?: LKBackspaces;
```

Comment:
This should probably have been in its own PR -- removal of backspaces is not related to adding of disps!

Reply:
Good point. Willl split